src/Akeneo/Connectivity/Connection/back/Domain/Apps/Persistence/Query/FindAllConnectedAppsQueryInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+declare(strict_types=1);
+
+namespace Akeneo\Connectivity\Connection\Domain\Apps\Persistence\Query;
+
+use Akeneo\Connectivity\Connection\Domain\Apps\Model\ConnectedApp;
+
+interface FindAllConnectedAppsQueryInterface
```

Comment:
```suggestion
interface GetConnectedAppsQueryInterface
```
Existing interfaces in the same namespace are using `get`, `delete` `save` `create`
same comment for find* queries

Reply:
Seen IRL, we'll see later